pkg/bff/models/v1/errors.go

```diff
@@ -1,10 +1,67 @@
 package models
 
-import "errors"
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
 
 var (
-	ErrMissingField        = errors.New("missing required field")
-	ErrInvalidCollaborator = errors.New("collaborator record is invalid")
-	ErrCollaboratorExists  = errors.New("collaborator already exists in organization")
-	ErrMaxCollaborators    = errors.New("maximum number of collaborators reached")
+	ErrMissingField          = errors.New("this field is required")
+	ErrInvalidCollaborator   = errors.New("collaborator record is invalid")
+	ErrCollaboratorExists    = errors.New("collaborator already exists in organization")
+	ErrMaxCollaborators      = errors.New("maximum number of collaborators reached")
+	ErrLegalPersonNameLength = errors.New("legal person name must be less than 100 characters")
+	ErrNoLegalNameIdentifier = errors.New("at least one legal name identifier is required")
+	ErrNoGeographicAddress   = errors.New("at least one geographic address is required")
+	ErrTooManyAddressLines   = errors.New("an address can have at most 7 address lines")
+	ErrInvalidAddress        = errors.New("address must have either address lines or street name and building number/name")
+	ErrNoAddressLines        = errors.New("geographic address requires at least one address line")
+	ErrInvalidCountry        = errors.New("country must be an ISO-3166-1 alpha-2 code")
+	ErrInvalidCustomerNumber = errors.New("customer number is optional but can be at most 50 characters")
+	ErrInvalidLegalNatID     = errors.New("national identifier type must be RAID, MISC, LEIX, or TXID")
+	ErrInvalidLEI            = errors.New("LEI identifier must not be longer than 35 characters")
+	ErrNoCountryNatID        = errors.New("country of issue must be empty for legal persons")
+	ErrLegalNatIDRequired    = errors.New("national identification is required to verify legal person")
+	ErrNoRAForLEIX           = errors.New("registration authority must be empty for identifier type LEI")
+	ErrRARequired            = errors.New("registration authority must be specified unless the identifier type is LEI")
 )
+
+type ValidationError struct {
+	Field string
+	Err   string
+	Index int
+}
+
+func (v *ValidationError) Error() string {
+	return fmt.Sprintf("invalid field %s: %s", v.Field, v.Err)
```

Comment:
The problem is that we don't know if index 0 is the index or just a field w/o indices. Since this is just the Error message I propose we leave it the same rather than trying to deal with that case.